libr/core/cmd_zign.c

```diff
@@ -686,9 +686,9 @@ static const char *getprefix(RSignType t) {
 	case R_SIGN_BBHASH:
 		return "bbhash";
 	default:
-		r_return_val_if_reached ("unknown_type");
+		break;
 	}
-	return "unk";
+	return NULL;
```

Comment:
why return NULL here? is there any caller that checks if the return value is NULL and behave different according to that?

Reply:
In theory this null is unreadhable code. So better have null deref than undefined behaviour or hiding the error with a valid string

Reply:
@as0ler can you check for the callers of this function and see if they handle null properly?